storey/aggregations.py

```diff
@@ -133,6 +134,9 @@ async def _do(self, event):
 
             event_timestamp = event.time
             if isinstance(event_timestamp, datetime):
+                if isinstance(event_timestamp, pd.Timestamp) and event_timestamp.tzinfo is None:
+                    local_time_zone = datetime.now().astimezone().tzinfo
+                    event_timestamp = event_timestamp.replace(tzinfo=local_time_zone)
```

Comment:
Just need a comment with the ticket number and maybe a short explanation.

Reply:
And it might make sense to extract a function from these 6 lines of code that are duplicated.